### ISSUE TITLE
UndefVarError in libgit2 error

### ISSUE THREAD

**Issue by vtjnash** [JuliaLang/julia#16337]
``` julia
=> Package RunTests cannot be updated.
GitError(Code:ERROR, Class:Net, Unsupported URL protocol)

=> Package MbedTLS cannot be updated.
UndefVarError: repo_ptr_ptr not defined
 in upstream(::Base.LibGit2.GitReference) at ./libgit2/reference.jl:141
```

I assume this should be `ref_ptr_ptr`?

**Comment by tkelman** [comment]
Looks like, cc @kshyatt ref #15662. Was probably getting swallowed by an over-aggressive try catch until recent changes?

**Comment by kshyatt** [comment]
Are you saying I caused this, or my tests exposed it? What is the appropriate level of shame I should be feeling right now?

**Comment by tkelman** [comment]
Unless Jameson has some local modifications to anything libgit2-related, very minor - just a typo, and it wasn't causing a test failure because it was concealed by catching too many exceptions elsewhere.

**Comment by wildart** [comment]
Yes, it's a typo. I'll fix this.

**Comment by wildart** [comment]
@vtjnash What is the content of `.git/config` for MbedTLS repo?

**Comment by tkelman** [comment]
any remaining typos that we know of getting suppressed by inappropriate try-catches, or was this closed by https://github.com/JuliaLang/julia/pull/16343?

**Comment by wildart** [comment]
yes, it's fixed by #16343

**Comment by KristofferC** [comment]
Everything in here seem fixed.
